examples/decoding/plot_decoding_xdawn_meg.py

```diff
@@ -0,0 +1,90 @@
+"""
+=============================
+ XDAWN Decoding From MEG data
+=============================
+
+ERF decoding with Xdawn. For each event type, a set of spatial Xdawn filters
+are trained and apply on the signal. Channels are concatenated and rescaled to
+create features vectors that will be fed into a Logistic Regression.
+"""
+# Authors: Alexandre Barachant <alexandre.barachant@gmail.com>
+#
+# License: BSD (3-clause)
+
+
+import mne
+from mne import io
+from mne.datasets import sample
+from mne.preprocessing.xdawn import Xdawn
+from mne.decoding import ConcatenateChannels
+
+from sklearn.cross_validation import StratifiedKFold
+from sklearn.pipeline import make_pipeline
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import classification_report, confusion_matrix
+from sklearn.preprocessing import MinMaxScaler
+import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
import order is not correct. Start from most general to local ie from np, plt to mne